compiler/tflchef/core/src/ModelChef.cpp

```diff
@@ -480,11 +482,23 @@ template <typename T> void cook_graph(const T &graph, CookParams &cp)
     // Create Operator
     tflite::OperatorBuilder op_builder{*flatbuffer_builder};
 
-    // Get operator code index from builtin_code_set with assumption, order of
-    // builtin_code_set is same as that of code_vec
+    // Note that opcode_index is an index into the operator_codes vector.
+    // operator_codes consists of buildtin_code and custom_code, which is inserted sequentially.
+    uint32_t opcode_index = 0;
     auto op_it = builtin_code_map.find(op_chef->code());
-    assert(op_it != builtin_code_map.end());
-    uint32_t opcode_index = std::distance(builtin_code_map.begin(), op_it);
+    // builtin operator
+    if (op_it != builtin_code_map.end())
+    {
+      opcode_index = std::distance(builtin_code_map.begin(), op_it);
+    }
+    // custom operator
+    else
+    {
+      auto op_it = std::find(custom_code_vec.begin(), custom_code_vec.end(), operation.type());
+      assert(op_it != custom_code_vec.end());
+      opcode_index += builtin_code_map.size();
+      opcode_index += std::distance(custom_code_vec.begin(), op_it);
```

Comment:
Um? Can you explain why we add both?

Reply:
> opcode_index is an index into the operator_codes vector.
> operator_codes consists of buildtin_code and custom_code, which is inserted sequentially.

`opcode_index` size is `builtin_code_map` size + `custom_code_vec` size. Since custom codes is preceded by builtin codes, the offset of custom operator's `opcode_index` is `builtin_code_map` size.

Reply:
OK, thanks for the explanation;
Then is the code guarantee that `builtin_code_map` WILL NOT change when this block executes?

And can you use like this?
```
      opcode_index = builtin_code_map.size();
      opcode_index += std::distance(custom_code_vec.begin(), op_it);
```

Reply:
@seanshpark I think `const` keyword would help.

Reply:
> I think const keyword would help.

Um? I don't get what const has go to do with guarantee ...

If this code block executes twice and in between `builtin_code_map` is increased then I code may not work correctly...

Reply:
@seanshpark 
`builtin_code_map` and `custom_code_vec` are not changed in `cook_graph` function. Could you explain the situation where they might be increased?

Reply:
> Could you explain the situation where they might be increased?

I was just concerned.

> builtin_code_map and custom_code_vec are not changed in cook_graph function. 

Then it's OK.